modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS). You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+.Procedure
+
+. Create a `Secret` YAML file that uses the `aws_access_key_id` and `aws_secret_access_key` fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+----
+
+
+. Create the secret. For example:
++
+[source,terminal]
+----
+$ oc apply -f cw-secret.yaml
+----
+
+
+. Create a `ClusterLogForwarder` CR YAML file that specifies the name of the secret. For example:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: cw <3>
+     type: cloudwatch <4>
+     cloudwatch:
+       groupBy: logType <5>
+       region: us-east-2 <6>
+     secret:
+        name: cw-secret <7>
+  pipelines:
+    - name: infra-logs <8>
+      inputRefs: <9>
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw <10>
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> Optional: Specify the option CloudWatch uses to group the logs. If you omit `groupBy`, its default value is `logType`.
+<6> Specify the AWS region.
+<7> Specify the name of the secret that contains your AWS credentials, which you created earlier.
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+.Example: How various configurations of the ClusterLogForwarder affect CloudWatch
+
+Suppose that you run an {product-title} cluster on AWS named `mycluster` and the value of its `infrastructureName` is `mycluster-7977k`.
```

Comment:
I'm reading this sentence like this:

If you ran an {product-title} cluster on AWS named myclusterand the value of itsinfrastructureNameismycluster-7977k, it would look like the following:`

Is this what you meant? If so, rephrasing it to something like I did above would better communicate that you're giving an example :)

Reply:
That's a good point, I don't think it's important at all that the OCP cluster is running on AWS or elsewhere.